src/sp_repo_review/checks/general.py

```diff
@@ -50,7 +50,12 @@ class PY003(General):
     @staticmethod
     def check(package: Traversable) -> bool:
         "Projects must have a license"
-        return len([p for p in package.iterdir() if "LICENSE" in p.name]) > 0
+        spellings = ("LICENSE", "LICENCE", "COPYING")
+        return any(
+            p
+            for p in package.iterdir()
+            if any(spelling in p.name for spelling in spellings)
+        )
```

Comment:
```suggestion
        return any(
            any(spelling in p.name for spelling in spellings)
            for p in package.iterdir()
        )
```

We don't want to check "any" on `p`, we want to check any on the bool value, I believe.

Reply:
```suggestion
        return any(s in p.name for p in package.iterdir() for s in spellings)
```

Actually, I think you can flatten it further, like this.

Reply:
I think they are equivalent, as any path would be truthy, but your way is clearer.

Reply:
Ahh, so much for being a *comprehension*.